custom_model_runner/requirements.txt

```diff
@@ -26,3 +26,4 @@ termcolor
 packaging
 markupsafe<=2.1.3
 pydantic==2.9.2
+datarobot-storage>=0.0.0,<1.0.0
```

Comment:
Curious about the reason to pin this to `0.*` versions specifically. Does it have something to do with artifactory releases vs. OSS releases? The reason I'm asking is Buzok is at `datarobot-storage==2.0.3` and we can't bump DRUM to the latest version because of this conflict.

Reply:
Sorry, I should probably rather ask in #drum in case we need the input of related teams on this.